lib/katello/permissions/capsule_content_permissions.rb

```diff
@@ -10,7 +10,7 @@
 
   permission :view_capsule_content,
             {
-              'smart_proxies' => [:pulp_storage, :pulp_status, :show_with_content]
+              'smart_proxies' => [:capsule_puppet_path, :pulp_storage, :pulp_status, :show_with_content]
```

Comment:
Is this leftover? Don't see a smart_proxies#capsule_puppet_path controller method